templates/v2/user_creation/components/_permissions_checkboxes.html.twig

```diff
@@ -6,44 +6,50 @@
         value: constant('App\\Entity\\MembreCentre::DEFAULT_PERMISSION_CREATE_BENEFICIARIES'),
         icon: 'user-plus',
         isOwned: true,
+        isUpdatable: false,
     },
     {
         name: 'beneficiaries_management'|trans,
         value: constant('App\\Entity\\MembreCentre::MANAGE_BENEFICIARIES_PERMISSION'),
         icon: 'user',
         isOwned: userRelay and userRelay.canManageBeneficiaries,
+        isUpdatable: is_granted('MANAGE_BENEFICIARIES', relay),
     },
     {
         name: 'pro_management'|trans,
         value: constant('App\\Entity\\MembreCentre::MANAGE_PROS_PERMISSION'),
         icon: 'user-cog',
         isOwned: userRelay and userRelay.canManageProfessionals,
+        isUpdatable: is_granted('MANAGE_PRO', relay),
     },
 ] %}
 
 {% for permission in permissions %}
-    {% set isDefaultPermission = permission.value == constant('App\\Entity\\MembreCentre::DEFAULT_PERMISSION_CREATE_BENEFICIARIES')%}
-    {% set isDisabled = not hasValidLinkToRelay or isDefaultPermission %}
-    <div class="d-flex mt-2 ms-md-5"
-            {% if isDisabled %}
-                data-toggle="tooltip"
-                data-bs-placement="right"
-                data-bs-custom-class="blue-tooltip"
-                title="{{ isDefaultPermission ? 'you_can_not_update_default_permission'|trans : 'wait_for_affiliation_validation_to_update_permission'|trans }}"
-            {% endif %}
-    >
-        <a
-                class="d-flex align-items-center justify-content-center btn btn-light border-primary {{ permission.isOwned ? activeClasses : inactiveClasses }} {{ isDisabled ? 'disabled' : '' }}"
-                style="height: 30px; width: 30px"
-                {{ stimulus_action('custom-checkbox', 'click', null, {
-                    path: path('toggle_pro_permission', {id: user.subjectMembre.id, relay: relay.id, permission: permission.value})
-                }) }}
+    {% set isDefaultPermission = permission.value == constant('App\\Entity\\MembreCentre::DEFAULT_PERMISSION_CREATE_BENEFICIARIES') %}
```

Comment:
Maybe add parenthesis to make it easier to read ? `  {% set isDefaultPermission = (permission.value == constant('App\\Entity\\MembreCentre::DEFAULT_PERMISSION_CREATE_BENEFICIARIES'))%}`